index.html

```diff
@@ -424,22 +423,25 @@ <h3>Signature Sets</h3>
         </p>
         <pre class="example highlight" title="A signature set in a Linked Data document">
 {
-  "@context": "https://w3id.org/identity/v1",
+  "@context": "https://www.w3.org/2018/credentials/examples/v1",
```

Comment:
This needs to have the base context added here.

Reply:
So, the reason it doesn't have a `https://www.w3.org/2018/credentials/v1` base context is that it's not a VC, it's just a random Hello World doc. Should I add it anyway?